### PR TITLE
LPS-67499

### PR DESCRIPTION
**Pull request by mhan810**

### CODE REVIEW

**Comment by mhan810** [comment]
@drewbrokke sorry this took so long.  Too many balls in the air.

The most interesting changes are in:
https://github.com/drewbrokke/liferay-portal/pull/233/commits/6b9ecab864928cd97f9cd71510b5e6e6fa55786f - we should use the Bundle to retrieve resources whenever possible.

https://github.com/drewbrokke/liferay-portal/pull/233/commits/386884500d440d3bd38f6fbdcb595e61446f82c3 - try to avoid using the Configurable object if possible.  This has performance overhead due to reflection.

https://github.com/drewbrokke/liferay-portal/pull/233/commits/780a0efc115caad8329afc352faed234e9014862 - can you check the logic here?  There were inconsistencies between how we check an image's companyId from everything else.  I made the image companyId check consistent with everyone else.

File: portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java [hunk 1]
```diff
@@ -302,8 +302,7 @@ else if (PATH_PORTLET_FILE_ENTRY.equals(pathArray[0])) {
 
 					long imageCompanyId = image.getCompanyId();
 
-					if ((imageCompanyId != userCompanyId) &&
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java]: The idea with this is that we've already checked the user's companyId, so there would be no need to check the image companyId if it's the same one as the user.

Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java]: I'm not opposed to removing this, though.  There's no harm in it, just possibly an extra check.

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by mhan810** [comment]
Got it perhaps put in a comment right above line to clarify?  I was
concerned from a maintenance perspective.

Sent from my mobile device.

**Comment by drewbrokke** [comment]
Pull request submitted to brianchandotcom#43210 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**